includes/class-sensei-question.php

```diff
@@ -1156,10 +1149,10 @@ public static function get_template_data( $question_id, $quiz_id ) {
 		$data['question_right_answer']  = get_post_meta( $question_id, '_question_right_answer', true );
 		$data['question_wrong_answers'] = get_post_meta( $question_id, '_question_wrong_answers', true );
 		$data['user_answer_entry']      = Sensei()->quiz->get_user_question_answer( $lesson_id, $question_id, get_current_user_id() );
-		$data['lesson_completed']       = Sensei_Utils::user_completed_course( $lesson_id, get_current_user_id() );
+		$data['lesson_completed']       = Sensei_Utils::user_completed_lesson( $lesson_id, get_current_user_id() );
 		$data['quiz_grade_type']        = get_post_meta( $quiz_id, '_quiz_grade_type', true );
 		$data['reset_quiz_allowed']     = $reset_allowed;
-		$data['lesson_complete']        = $user_lesson_complete;
+		$data['quiz_is_completed']      = Sensei_Quiz::is_quiz_completed();
```

Comment:
Do you think it makes sense to pass the `$quiz_id` variable when calling the `is_quiz_completed` method?

Reply:
How I can know what can be used externally?  @m1r0

Reply:
Oh, my comment was in the wrong place, please ignore it.

I agree with quiz_id, I am going to update.